install_config/upgrading/automated_upgrades.adoc

```diff
@@ -232,6 +232,15 @@ endif::[]
 :sect: automated
 include::install_config/upgrading/manual_upgrades.adoc[tag=30to31updatingcerts]
 
+[[automated-upgrading-efk-logging-stack]]
+== Upgrading the EFK Logging Stack
+
+If you have previously link:../../install_config/aggregate_logging.html[deployed
+the EFK logging stack] and want to upgrade to the latest logging component
+images, the steps must be performed manually as shown in
+link:../../install_config/upgrading/manual_upgrades.html#manual-upgrading-efk-logging-stack[Manual
+Upgrades].
+
```

Comment:
@dgoodwin Added a pointed from the Automated Upgrades to the manual steps for now, which we can update if the playbooks are able to handle this in later releases.